developer_setup.md

```diff
@@ -83,17 +83,34 @@ As per 20+, with the following changes:
   brew install git
   brew install memcached
   brew install postgresql
+  brew install graphviz
   ```
 
+If `bundle install` (when running `bin/setup` below) fails in installing the `eventmachine` gem,
```

Comment:
Would you consider moving this to below the `bin/setup` step? If people have a problem with that, they will look at notes in that section, not up here.

Then it reads something like

"If bin/setup fails on installing the `evenmachine` gem, run the following command and then retry bundle install: